runtime/doc/treesitter.txt

```diff
@@ -940,8 +939,8 @@ lint({buf}, {opts})                              *vim.treesitter.query.lint()*
     for the `lua` language will be used.
 
     Parameters: ~
-      • {buf}   (integer) Buffer handle
-      • {opts}  (QueryLinterOpts|nil) Optional keyword arguments:
+      • {buf}   (`integer`) Buffer handle
+      • {opts}  (`table?`) Optional keyword arguments:
```

Comment:
Loss of typing?

Reply:
I did this on purpose since the type isn't used anywhere else.

Basically, if you do:
```
--- @param name some.complicated.type (table) desc
```

It will render the type as `table` instead of `some.complcated.type`.

Reply:
Ah, but these aren't documented, so it's either weird type here or loss of typing in luals.